apps/src/templates/projects/ProjectCard.jsx

```diff
@@ -154,6 +153,11 @@ export default class ProjectCard extends React.Component {
               />
             </div>
           )}
+          {shouldShowPublishedAt && projectData.isFeatured && (
```

Comment:
do we still want to check `shouldShowPublishedAt` here? Would we ever want to show the Featured Project label for projects where we wouldn't show when they were published?

If we still want to do this check, I think it would be a little clearer to name it something else, because it seems confusing to require `showPublishedAt` to be true and then not actually show when the project was published.

Reply:
I think as-designed, we'll only want to show this label if we would otherwise be showing the publish date (in the public gallery and when displaying detail view). We don't want to display it on the "My dashboard" list of tiles, for instance. So I think the other conditions inside this flag still apply and it's currently used to determine how the card's styled (just padding for 3 lines vs. 2) so I'll rename and adjust the conditions a bit!